manual/en/fundamentals.html

```diff
@@ -422,12 +422,12 @@ <h3>es6 modules, three.js, and folder structure</h3>
 libraries they'll both reference the same <code class="notranslate" translate="no">three.module.js</code> file.
 </p>
 <pre class="prettyprint">import * as THREE from './someFolder/build/three.module.js';
-import {OrbitControls} from './someFolder/examples/jsm/controls/OrbitControls.js';
+import {OrbitControls} from './someFolder/addons/controls/OrbitControls.js';
```

Comment:
This file needs a heavier rewriting

Reply:
Something like this: https://github.com/mrdoob/three.js/pull/24888/commits/b6e279709ee57d0a4b3707e82d7d0ceede8276a5

Reply:
Possibly something about Node/NPM usage also should be added there.